.eslintrc.js

```diff
@@ -0,0 +1,47 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: [
+    'plugin:import/recommended',
+    'plugin:promise/recommended',
+    'standard',
+    'standard-react',
```

Comment:
Just as a note for something we could do everywhere and not necessarily in this PR: I wonder if we could replace all of this by `eslint:recommended` and `plugin:react/recommended`?

Reply:
Let's have this discussion elsewhere and keep it like this for now; `standard` brings a few more plugins / rules (promise and import) than the baseline recommended settings.

Reply:
👍